itou/www/dashboard/views.py

```diff
@@ -112,7 +112,9 @@ def dashboard(request, template_name="dashboard/dashboard.html"):
         "can_create_siae_antenna": False,
         "can_show_financial_annexes": False,
         "can_show_employee_records": False,
-        "can_view_gps_card": is_allowed_to_use_gps(request.user),
+        "can_view_gps_card": (
+            is_allowed_to_use_gps(request.user) and getattr(request.current_organization, "department", None) != "30"
```

Comment:
Peut-être sortir un helper pour lier le check du menu avec celui du dashboard?

Genre
```python
def in_gard(request):
    return getattr(request.current_organization, "department", None) == "30"
```

Sur le dashboard, `not in_gard(request)`, et dans le menu `in_gard(request)`. L’idée étant de maintenir la cohérence si jamais on ouvre à d’autres départements et montrer le lien logique entre le dashboard et le menu.